README.yaml

```diff
@@ -140,6 +140,24 @@ usage: |2-
   ```
 
   The `container_image` in the `container_definition` module is the Docker image used to start a container.
+  
+  `container_definition` is a JSON-encoded list of container definitions. Normally you would place only one container definition here as a single
+  definition object in a list, i.e. `[{"name": "sweetops"}]`. However, there might be situations where more than one container per task
```

Comment:
>Normally you would place only one container definition here as a single definition object in a list

It does not have to be in a list if it's a single task definition.

> i.e. `[{"name": "sweetops"}]`

This is an invalid container definition json. It would be better to use a module reference here or just omit this example... fyi in this case you'd use `e.g.` instead of `i.e.` since the former is "for example" whereas the other is "in other words"